core/trino-main/src/main/java/io/trino/server/security/PasswordAuthenticatorManager.java

```diff
@@ -67,39 +76,42 @@ public void loadPasswordAuthenticator()
             return;
         }
 
-        File configFile = CONFIG_FILE.getAbsoluteFile();
-        try {
-            Map<String, String> properties = new HashMap<>(loadPropertiesFrom(configFile.getPath()));
+        ImmutableList.Builder<PasswordAuthenticator> authenticators = ImmutableList.builder();
+        for (File configFile : configFiles) {
+            configFile = configFile.getAbsoluteFile();
+            try {
```

Comment:
Can you try moving this to a method `loadAuthenticator(File)` directly below so that the indentation doesn't change?